src/ui/profile_preferences.ui

```diff
@@ -1341,124 +1341,90 @@
        <string>Mapper</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_24">
-       <item row="1" column="0">
-        <widget class="QGroupBox" name="downloadMapOptions">
+       <item row="4" column="0">
```

Comment:
After editing with the QtDesigner plug-in (or the stand-alone one) you may need to re-edit the file with the text editor and correct the ordering of the `<item row=X column=Y>` items (in the built-in editor you can collapse the items down to just show this type of line; which makes it easier to cut and paste to reorder them by row and then column.  It does not effect the operational use of the `.ui` file but it can improve the Signal to Noise ratio in git differences if we keep/get `QGridLayouts` ordered.

The requirement to get these parts of these files in a sensible order is the subject of #814...! 😀